README.md

```diff
@@ -1,3 +1,7 @@
+# Deprecation Notice
+
+This Action is deprecated. Instead, you can set up ongoing configuration file import using [Azure CLI](https://learn.microsoft.com/en-us/azure/azure-app-configuration/push-kv-github-action) GitHub action, which offers full capabilities for file importing to your App Configuration store. 

```

Comment:
Change the wording a little bit.

This GitHub action has been deprecated. The Azure CLI for Azure App Configuration provides full capabilities for importing configuration files into Azure App Configuration. Follow [these instructions](https://learn.microsoft.com/en-us/azure/azure-app-configuration/push-kv-github-action) to set it up in your GitHub actions.

Reply:
This has been updated